helm_deploy/licences/templates/_envs.tpl

```diff
@@ -153,4 +153,10 @@ env:
 
   - name: REDIS_TLS_ENABLED
     value: "true"
+
+  - name: HDC_URL
```

Comment:
The HTML templates for forms and licences include links to CSS files served from within the public folded of licences.  HDC_URL is the prefix for those links.